src/include/mallocMC/creationPolicies/Scatter_impl.hpp

```diff
@@ -75,7 +75,7 @@ namespace ScatterKernelDetail{
   {
     public:
       typedef T_Config  HeapProperties;
```

Comment:
Somehow clang-tidy missed to change this. Could you maybe grep for `typedef` and adjust this and other lines by hand.

Reply:
thx, done!